### ISSUE TITLE
From yesterday onwards I'm also facing the same issue unable to login to postman account when click on sign in.

### ISSUE THREAD

**Issue by shaibaz02** [a85/POSTMan-Chrome-Extension#10109]
From yesterday onwards I'm also facing the same issue unable to login to postman account when click on sign in.
![image](https://user-images.githubusercontent.com/79316992/125190077-fdeb7c80-e258-11eb-8bbe-363677cddfe1.png)

_Originally posted by @shaibaz02 in https://github.com/postmanlabs/postman-app-support/issues/7171#issuecomment-877769629_

**Comment by postman-staff** [comment]
Hi there,

Thanks for reaching out and sorry to see you are facing issues logging into your Postman account.

To troubleshoot further we will need app logs and other details. If you are still facing the same issue then we would like to investigate further. Could you open a new ticket on our Support channel using this link - https://www.postman.com/support/, please? Also, do add this GitHub issue number and your GitHub username in the request description to make it easy for us to track.

Meanwhile, to get you unblocked could navigate the below links to check if Postman on Web works as expected for you? Postman on Web has the same UI & features as that of the desktop app, so feel free to continue using it, if that works as expected.

- Postman on Web for Team users - https://go.postman.co/
- Postman on Web for Single/non-team users - https://web.postman.co/

We will wait to hear back from you on our support channel.

via @Swapnil-getpostman | Postman Support

**Comment by shaibaz02** [comment]
Hi Swapnil,
As you have said to check with these two links is not working on my wifi network but when I switched with my mobile data everything goes to normal and everything working fine so I can't figure out what is the problem. I google the problem but could not figure out I not be able to authenticate postman links using wifi network but changing to the mobile network works fine. So can you tell me what is the problem?
![image](https://user-images.githubusercontent.com/79316992/125242817-cccb8480-e30a-11eb-9e8c-76cbc8d6a532.png)

**Comment by postman-staff** [comment]
Hi there,

Thanks for writing back.

Just want to check - are you behind any Corporate proxy for having any firewall or VPN installed over your system?

May I know have you tried, renaming the Postman installation folder? It generally resolves this issue and you can find full instructions for doing so in the below article. Hope this helps!

- https://support.postman.com/hc/en-us/articles/360025359014#h_b38048ee-6505-4f83-a0f0-5d4f51fcb6d5

Let me know if you still face the issue after renaming the installation folder. Looking forward to your response.

via @Swapnil-getpostman | Postman Support

**Comment by shaibaz02** [comment]
Hi Swapnil,
Thanks for your effort in resolving my issue but I figured that the issue is with my wifi ISP and hence I sorted it out with help of them and now the postman working fine. Once again thanks.

**Comment by postman-staff** [comment]
That's great and thanks for the update.

I'm going ahead and closing this conversation for now and you can also open a new ticket on our Support channel using this link - https://www.postman.com/support/. Hope this helps!

via @Swapnil-getpostman | Postman Support
